src/utility.js

```diff
@@ -453,6 +453,55 @@ function createItem(args, logger, notifier, requestKeys, lambdaContext) {
   return item;
 }
 
+var TELEMETRY_TYPES = ['log', 'network', 'dom', 'navigation', 'error', 'manual'];
+var TELEMETRY_LEVELS = ['critical', 'error', 'warning', 'info', 'debug'];
+
+function arrayIncludes(arr, val) {
+  var len = arr.length;
+  var k = 0;
+
+  while (k < len) {
```

Comment:
Is there a reason for a while loop where and not a for loop?

Reply:
No, it's arbitrary. Is there a reason to prefer a for loop?

Reply:
Not really, just consistency. This would be only the second while loop in this file and the only other one in `parseUri` could use a cleanup. I mean they are the exact same when it comes to execution, but I was just surprised to see a while loop in this function and a for loop in the next function within the same PR.

Reply:
Same with `++i` vs `i++`, in the loop postamble they are the same but throughout this codebase we mix them a bit, I'd other stick with one. I can go through after this PR and clean up all my weird style nits.

Reply:
All makes sense. I'll take care of these before merge.

Reply:
I went ahead and also cleaned up `parseUri`, and converted the few postfix increment/decrements in utility.js to prefix.